core/src/main/cfml/context/admin/overview.cfm

```diff
@@ -773,7 +773,10 @@ Error Output --->
 <cfscript>
 	function getJavaVersion() {
 		var verArr=listToArray(server.java.version,'.');
-		if(verArr[1]>2) return verArr[1];
-		return verArr[2];
```

Comment:
Wasn't the 2nd return in the old code effectively an "else", similar to what the new code is doing? Sorry maybe it's too late in the evening for me but I'm not seeing how the new code fixes it. I would have expected an `elseif (ArrayLen(verArr) GT 1)` or something like that.

Reply:
@kabutotx  I think this will do it.

`	function getJavaVersion() {
		var verArr=listToArray(server.java.version,'.');
		if(verArr[1]>2) {
			return verArr[1];
		} elseif (verArr.len() GT 1) {
			return verArr[2];
		} else {
		    return val(server.java.version);
		}
	}`